pkg/test/server.go

```diff
@@ -17,13 +17,23 @@ limitations under the License.
 package test
 
 import (
+	"os"
 	"time"
 
+	"github.com/kubernetes-sigs/kubebuilder/pkg/install"
 	extensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	"k8s.io/client-go/rest"
-
 	"sigs.k8s.io/testing_frameworks/integration"
-	"github.com/kubernetes-sigs/kubebuilder/pkg/install"
+)
+
+// Default binary path for test framework
+const (
+	env_KubeCtl = "TEST_ASSET_KUBECTL"
+	env_Api     = "TEST_ASSET_KUBE_APISERVER"
+	env_Etcd    = "TEST_ASSET_ETCD"
```

Comment:
Lets rename these to camelcase version and have them in a separate block from defaultBinPaths.

env_KubCtl --> envKubectlBin
env_Api --> envKubeAPIServerBin
env_Etcd --> envEtcdBin

Reply:
Done